libs/agno/agno/agent/agent.py

```diff
@@ -264,6 +265,8 @@ class Agent:
     knowledge_retriever: Optional[Callable[..., Optional[List[Union[Dict, str]]]]] = None
     references_format: Literal["json", "yaml"] = "json"
 
+    skills: Optional[Skills] = None
```

Comment:
Let's design it so it's `Optional[List[Skill]]`? one individual Skill makes sense